CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8.10)
 project(vr-update CXX)
 
-set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD 23)
```

Comment:
This can be issue with older version Openbmc , which doen't support GCC version  with latest.